Task_TheFlexGame/index.html

```diff
@@ -1,15 +1,36 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Flex Game</title>
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-
-    <!-- Start coding from the lines below for FLEX GAME TASK; DO NOT TOUCH the code above -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Flex Game</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 0; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>
+
+    <!-- Start coding from the lines below for FLEX GAME TASK; DO NOT TOUCH the code above -->
+      <!--for manu-->
+    <div class="manu">
+        <div class="manu-box">Manu-1</div>
+        <div class="manu-box">Manu-2</div>
+        <div class="manu-box">Manu-3</div>

```

Comment:
There's an extra whitespace after the closing tag of the 'div' element. Remove it for consistency.